libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/avatar/Avatar.kt

```diff
@@ -85,12 +88,13 @@ private fun ImageAvatar(
 @Composable
 private fun InitialsAvatar(
     avatarData: AvatarData,
+    avatarColors: AvatarColors?,
     modifier: Modifier = Modifier,
 ) {
     // Use temporary color for default avatar background
     val avatarColor = ElementTheme.colors.bgActionPrimaryDisabled
```

Comment:
Maybe rename this to `fallbackAvatarColor` and modify the comment above?

Reply:
Actually, I'm not sure if I should just remove the parameter `avatarColors`, since it is only used by the `InitialsAvatar` composable... So it could be computed only in this case. But since it's already computed for the sender display name, I did not want to compute it twice so I have added a parameter... What would you do?

Reply:
Actually we cannot compute the colors here, since InitialAvatar can also used for the room avatar, and this PR is only about user avatar.

Reply:
Are you sure? On the latest EW nightly I see plenty of rooms with no avatars defaulting to these kind of avatar colors:

<img width="135" alt="image" src="https://github.com/vector-im/element-x-android/assets/480955/4771aada-2048-4dfa-8f00-2de166b59b39">

I'll ask in the Compound room in any case.

Reply:
That said, I doesn't need to be done in this PR.

Reply:
I see, I will merge and apply the changes to rooms in a dedicated PR.